runtime/caml/roots.h

```diff
@@ -21,16 +21,25 @@
 #include "misc.h"
 #include "domain.h"
 
+enum scanning_action_flags {
+  SCANNING_ONLY_YOUNG_VALUES = 1, // action is a no-op outside the minor heap
+};
```

Comment:
```suggestion
typedef enum {
  SCANNING_GENERIC = 0, // no special property
  SCANNING_ONLY_YOUNG_VALUES = 1, // action is a no-op outside the minor heap
} scanning_action_flags;
```
How about a typedef to omit `enum` in signatures? (like other typedef enums in the runtime)

Reply:
I went for your `typedef` suggestion. (It's a question of style and I have a mild preference for what I had written, but it's good to put more weight in other people's expressed preference, and indeed it seems that this style dominates in existing code.)